integration/node_app_test.go

```diff
@@ -77,6 +85,11 @@ func testNodeApp(t *testing.T, context spec.G, it spec.S) {
 			Expect(err).NotTo(HaveOccurred())
 
 			Eventually(container).Should(Serve(ContainSubstring("Loading...")).OnPort(8080))
+
+			// check an SBOM file
+			contents, err := os.ReadFile(filepath.Join(sbomDir, "sbom", "launch", strings.ReplaceAll(settings.BuildpackInfo.Buildpack.ID, "/", "_"), "sbom.cdx.json"))
+			Expect(err).NotTo(HaveOccurred())
+			Expect(string(contents)).To(ContainSubstring(`"name": "System.Text.Encoding"`))
```

Comment:
In the case of an app that uses node, does the syft-generated SBOM include `node_modules`? If not, let's file an issue to explore if we can either a) add that to syft or b) add another SBOM generation step to this buildpack to get those `node_modules` included.

Reply:
I am not sure that the app keeps the `node_modules` around after the publish process as it is in the list of excluded files from the final published application. So we would have to find a way to scan mid build if we wanted that information.

There is an embedded frontend application that we could potentially get if we scanned before publish and set it as the build time SBOM but I am not sure if we would get a large amount of information from a `package-lock.json` so we would have to test it.

Reply:
Did a quick test where the `node_modules` are not removed from the final app and there is nothing in the Syft output regarding the `node_modules`. I think that this would take some much deeper testing to get right though and feels like an additional bonus.

Reply:
I am willing to to open this up as an issue for down the road though.

Reply:
Seems like removing the node modules from the final app could also be a neat feature for trimming down the image. All in all, seems like our .NET + Nodejs user experience could be more mature overall. I'm not so sure how common a workflow it is. Maybe we consult with some .NET SMEs about it in the future?

Reply:
Filed an issue to document this: https://github.com/paketo-buildpacks/dotnet-execute/issues/320